activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
Given the previous number of entries in the sidebar, I think most people would type `HashWith..` in the search bar but even though the guides directly refer to `ActiveSupport::HWIA` most of the time, I may be wrong so I can add it back if you want. :-)